### ISSUE TITLE
Difference between .score and .get_param_scores()

### ISSUE THREAD

**Issue by carmkhoo** [cgnorthcutt/hypopt#24]
I am calling:

model = GridSearch(...)
model.fit(..., scoring='roc_auc')

I do have validation and test sets, the former of which is passed into model.fit(..). To get both validation and test ROC scores, I call model.score() on both my validation and test sets. 

Since I would like to keep track of the ROC returned by each set of hyperparameters, I am also saving the output from model.get_param_scores(). From my understanding, the first ROC from model.get_param_scores() should be identical to my validation ROC obtained from model.score() (since they are sorted in descending order). However, the values are completely different. 

**My question is**: Is model.score() actually using the specified metric (i.e. roc_auc) or is it only computing accuracy?

Thank you!
